astropy_helpers

```diff
@@ -1 +1 @@
-Subproject commit 3a1f7998e00d29db6a9e66465f21fd2d635f44cd
+Subproject commit 160b48587680e11f12b66a51641a2601b869d1fa
```

Comment:
@dpshelio: Ghosts of astropy helpers past, Leave me alone. I will fix this by tomorrow.